File: Python/Easy/cone_volume.py
# https://edabit.com/challenge/uXPF7XM4KoiiSnjuA
from math import pi
def cone_volume(h, r):
    return round(pi*(r**2)*(h)/3,2)

print(cone_volume(3, 2), 12.57)
print(cone_volume(15, 6), 565.49)
print(cone_volume(18, 0), 0)
print(cone_volume(100, 2), 418.88)
print(cone_volume(1, 1), 1.05)
print(cone_volume(0, 30), 0)

File: javascript/Medium/coneVolume.js
// https://edabit.com/challenge/P6XbTvWyHkrpPfhvu


function coneVolume(h, r) {
	return +(Math.PI*r**2*h/3).toFixed(2)
}


console.log(coneVolume(3, 2), 12.57)
console.log(coneVolume(15, 6), 565.49)
console.log(coneVolume(18, 0), 0)
console.log(coneVolume(100, 2), 418.88)
console.log(coneVolume(1, 1), 1.05)
console.log(coneVolume(0, 30), 0)